src/index.js

```diff
@@ -180,6 +180,10 @@ cronofy.prototype.profileInformation = function(){
 cronofy.prototype.readEvents = function(){
   var details = this._parseArguments(arguments, ["access_token"]);
 
+  if (details.options.next_page) {
+    return this._httpCall('GET', details.options.next_page, details.options, details.callback, ['access_token', 'next_page']);
+  }
+
   return this._httpGet(details.options.next_page || '/v1/events', details.options, details.callback);
```

Comment:
Could just be:

```js
this._httpGet('/v1/events', ...
```

As `details.options.next_page` will never be truthy at this point any more.